src/global/helpers/forward-event.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Forwards a change on the nested input element to the host element.
+ * In this way an event triggered in the ShadowDOM can cross its boundary and can be listened on the host component.
+ */
+export function forwardInnerEventToHost(event: Event, host: HTMLElement): void {
+  const eventConstructor = Object.getPrototypeOf(event).constructor;
+  const copiedEvent: Event = new eventConstructor(event.type, event);
+  host.dispatchEvent(copiedEvent);
+}
```

Comment:
```suggestion
/**
 * Forwards an event to the host element provided.
 * With this way, an event triggered in the ShadowDOM can cross its boundary and can be listened on the host component.
 */
export function forwardEventToHost(event: Event, host: HTMLElement): void {
  const eventConstructor = Object.getPrototypeOf(event).constructor;
  const copiedEvent: Event = new eventConstructor(event.type, event);
  host.dispatchEvent(copiedEvent);
}
```

Reply:
Done in [af92fa8](https://github.com/lyne-design-system/lyne-components/pull/1426/commits/af92fa81c5233debb291861981681a74de186fea)